packages/client/components/navigation/NavLink.tsx

```diff
@@ -0,0 +1,44 @@
+import { Link, Text, Box, HStack } from '@chakra-ui/react';
+import NextLink from 'next/link';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { IconProp } from '@fortawesome/fontawesome-svg-core';
+
+interface LinkProps {
+  path: string;
+  icon: IconProp;
+  title: string;
+}
+[];
```

Comment:
What's this lil guy for?

Reply:
Type checking the incoming props from the Hire sidebar like so

```
<NavLink
           path="/my_projects"
           icon={faClipboardList}
           title={t('components.hire_sidebar.my_projects')}
         />
   ```

Reply:
Or the Earn Sidebar

Reply:
I'm meaning the `[];` That's part of it?

Reply:
I was previously trying to map the Navlink props from an array of objects and I’m not well versed with ts so just forgot to remove them when I explicitly called the component